packages/ui5-middleware-fe-mockserver/README.md

```diff
@@ -86,8 +81,14 @@ Additional option are available either per service of for all services if define
 - logger: specify the logger to use, by default it will use the @ui5/logger
 - watch : toggle the watch mode, the mockserver will restart the service where data or metadata have changed
 - noETag : disable ETag support on metadata
-- strictKeyMode : disable the default "loose" mode for the key matching, you can try this if the mockserver returns too much data
-- contextBasedIsolation : enable the support of "tenants", by adding /tenant-xxx at the very start of your service call you will be able to work on tenant isolated data.
+- strictKeyMode : disable the default "loose" mode for the key matching.
+By default the mockserver will try many ways to match an entityset with its navigation property, which can result in some rather loose join of data. In case you have a very well defined mockdata and you feel the mockserver is returning too much data, you can enable this feature.
+- contextBasedIsolation : enable the support of "tenants".
+Tenants can be enabled either by using the ?sap-client in your application / request or by prepending /tenant-xxx at the very start of your service call.
+With tenants you will be able to work on tenant isolated data.
+
+Tenants will automatically try to fetch the json data from a file including the tenant ID in the name. 
```

Comment:
JS file is still needed if you want more options or if you want to mock actions and the like.
I'll update the wording for the json loading so that it's not confusing

Reply:
If nothing else needs to be customised, it would be nice if no JS file is necessary, only if custom loading or an action is necessary, right?

Reply:
No JS file is needed if that's the only customization required yes :)
I've included as much in DefiningMockData.md now